### DISCUSSION TITLE
CodeQL report visualization

### DISCUSSION THREAD

**Discussion by chaitanya-bojja-by** [community/community#25009] [Actions]
<p>hi Team,</p>
<p>I have added CodeQL for code scans as part of CI, how and where to visualize the report to everyone as its part of the security section where everyone does not have access to it.</p>
<p>I see 40 -50 violations as part of the scan I would like to show to the team like the sonarQube Dashboard.</p>

**Comment by BrightRan** [comment]
<p><a class="mention" href="/chaitanya-bojja-by">@chaitanya-bojja-by</a> ,</p>
<p>You can try using the GitHub App “<a href="https://github.com/marketplace/lgtm" rel="noopener nofollow ugc"><strong>LGTM</strong></a>” in your repository.</p>

**Comment by jhutchings1** [comment, marked as answer]
<p>Everyone with write access to your repository has access to the Code scanning section of the security tab. We hope this maps pretty closely to the folks you’d like to access this data!</p>
